tests/general/LicenseCheckBearTest.py

```diff
@@ -0,0 +1,67 @@
+import os
+from queue import Queue
+
+from bears.general.LicenseCheckBear import LicenseCheckBear
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.results.Result import Result
+from coalib.settings.Section import Section
+
+
+def get_testfile_path(name):
+    return os.path.join(os.path.dirname(__file__),
+                        'licensecheck_test_files',
+                        name)
+
+
+def load_testfile(name):
+    return open(get_testfile_path(name)).readlines()
+
+
+@generate_skip_decorator(LicenseCheckBear)
+class LicenseCheckBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.uut = LicenseCheckBear(Section('name'), Queue())
+
+    def test_license(self):
+        file_contents = load_testfile('mit_license.py')
+        self.check_results(
+            self.uut,
+            file_contents,
+            [],
+            filename=get_testfile_path('mit_license.py'))
+
+    def test_license_without_copyright(self):
+        file_contents = load_testfile('apache_license_without_copyright.py')
+        self.check_results(
+            self.uut,
+            file_contents,
+            [],
+            filename=get_testfile_path('apache_license_without_copyright.py'),
+            settings={'licensecheck_lines': 'int = 70',
+                      'licensecheck_tail': 'int = 0'})
+
+    def copyright_without_license(self):
+        file_contents = load_testfile('copyright_without_license.py')
+        self.check_results(
+            self.uut,
+            file_contents,
+            [Result.from_values('LicenseCheckBear',
+                                'No license found.',
+                                file=get_testfile_path('copyright_without_'
+                                                       'license.py'))],
+            filename=get_testfile_path('copyright_without_license.py'),
+            settings={'licensecheck_lines': 'int = 0',
+                      'licensecheck_tail': 'int = 0'})  # Parse entire file
+
+    def no_license(self):
+        file_contents = load_testfile('no_license.py')
+        self.check_results(
+            self.uut,
+            file_contents,
+            [Result.from_values('LicenseCheckBear',
+                                'No license found.',
+                                file=get_testfile_path('no_license.py'))],
+            filename=get_testfile_path('no_license.py'),
+            settings={'licensecheck_lines': 'int = 10'})
```

Comment:
Yes @Makman2 , you are absolutely correct, actually I had copied that statement from the docs and added quotes because of some error in haste. Have filed an issue for the same. :D